### ISSUE TITLE
TooltipClass option on directive doesn't work

### ISSUE THREAD

**Issue by chillyistkult** [benmarch/angular-ui-tour#38]
Unfortunately I have another issue with the tooltip class option on the ui-tour directive.

``` html
<div class="with-top-navbar"
     ui-tour
     ui-tour-scroll-offset="270"
     ui-tour-tooltip-class="'tour-tooltip'" //Does not work
     ui-tour-backdrop="true"
     ui-tour-on-ready="onTourReady(tour)"
     ui-tour-on-show="onTourStep(tour)"
     ui-tour-on-end="onTourEnd()"
     ui-tour-template-url="'app/app/shared/tutorial.html'">
    <header data-ng-include="'app/app/header/header.html'"></header>
    <messages class="messages-fixed-top"></messages>
    <div id="content" ui-view></div>
    <!--<footer id="footer" data-ng-include="'app/app/footer/footer.html'"></footer>-->
</div>
```

I can't even find any reference in the `angular-ui-tour.js` which is dealing with this property. Is it just missing or do I miss something?

**Comment by benmarch** [comment]
Hey @chillyistkult, sorry for these issues, it looks like I copied a lot of the documentation over from Angular Bootstrap Tour but didn't implement some of the features yet. This one will be a bit tricky, and I may need to change it from "tooltipClass" to just "class". I will have this one fixed by this time tomorrow. Let me know if you find any other issues and I can address those as well.

**Comment by benmarch** [comment]
Just released 0.5.2 and changed the option to "popupClass" which should work in the config provider, uiTour directive, and tourStep directive.

**Comment by benmarch** [comment]
Actually, use 0.5.3 instead, fixed another issue.
